docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
       - ./config:/mediawiki/config
       - ./images:/mediawiki/images
       - sitemap:/mediawiki/sitemap
+      - ./extensions-data:/mediawiki/extensions
```

Comment:
I think it's confusing to call this `/mediawiki/extensions` when the mount is called `extensions-data` and contains more than just extension data - it also contains extensions code!